modules/core/shared/src/main/scala/lucuma/core/enums/GuideProbe.scala

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.enums
+
+import lucuma.core.util.Enumerated
+
+/**
+ * Enumerated type for guide probe
+ */
+enum GuideProbe(val tag: String) extends Product with Serializable derives Enumerated {
```

Comment:
Do we need `Product` and `Serializable` on an enum?

Reply:
Nope, actually this was a test because the `Codec` test was failing. it seems circe tries to test serialization as well which we don't care.
I'll remove this